# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../beep/Makefile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- beep-1.2.2/Makefile.vanilla	2008-07-11 22:17:31.000000000 +0200
+++ beep-1.2.2/Makefile	2008-07-11 22:18:06.000000000 +0200
@@ -15,5 +13,5 @@
 
 install : 
 	cp ${EXEC_NAME} ${INSTALL_DIR}
-	rm -f /usr/man/man1/beep.1.bz2
+	rm -f ${MAN_DIR}/beep.1.bz2
 	cp ${MAN_FILE} ${MAN_DIR}
